tests/python/test_api/main.py

```diff
@@ -5,8 +5,8 @@
 It uses a lot of different aspects of the Rerun API in order to test it.
```

Comment:
The docstring for this file should be updated now that it is no longer an example, but a test!